src/ui/SearchAlerts/SearchAlerts.ts

```diff
@@ -47,37 +49,48 @@ export class SearchAlerts extends Component {
   static options: ISearchAlertsOptions = {
 
     /**
-     * Specifies whether to add a menu item in the {@link Settings} component to allow the user to manage their search
-     * alerts.
+     * Specifies whether to add the **Manage Alerts** item in the {@link Settings} menu to allow the end user to manage
+     * search alerts.
+     *
+     * Clicking the **Manage Alerts** item calls the {@link SearchAlerts.openPanel} method.
      *
      * Default value is `true`.
      */
     enableManagePanel: ComponentOptions.buildBooleanOption({ defaultValue: true }),
 
     /**
-     * Specifies whether to add a menu item in the {@link Settings} component to allow the user to follow the last
-     * query.
+     * Specifies whether to add the **Follow Query** item in the {@link Settings} menu to allow the end user to follow
+     * the last query.
+     *
+     * Clicking the **Follow Query** item calls the {@link SearchAlerts.followQuery} method.
      *
      * Default value is `true`.
      */
     enableFollowQuery: ComponentOptions.buildBooleanOption({ defaultValue: true }),
 
     /**
-     * Specifies the modifiedDateField to use when sending the followQuery subscription request.
+     * Specifies which field to use to represent the modification date when sending the
+     * {@link ISubscriptionQueryRequest}.
      *
      * Default value is `undefined`.
      */
     modifiedDateField: ComponentOptions.buildFieldOption(),
 
     /**
-     * Specifies whether to display info and error messages when search alerts actions are performed.
+     * Specifies whether to display info and error messages when performing search alerts actions.
+     *
+     * If this options is `true`, the SearchAlerts constructor will automatically instantiate a
+     * {@link SearchAlertsMessage} component and set it to the {@link SearchAlerts.message} attribute.
+     *
+     * See also {@link SearchAlerts.options.messageCloseDelay}.
      *
      * Default value is `true`.
      */
     enableMessage: ComponentOptions.buildBooleanOption({ defaultValue: true }),
 
     /**
-     * Specifies how long to display the search alert messages (in ms).
+     * If {@link SearchAlerts.options.enableMessage} is `true`, specifies how long to display the search alert messages
```

Comment:
I mentioned that the enableMessage has to be `true` because it seems to make sense, but the option does not actually require this. What do you think? Should we keep this comment? Should we maybe add a `depend: 'enableMessage'` to the option?